build/templates/status.inc.mako

```diff
@@ -11,7 +11,8 @@
 <%
 table_contents = []
 table_contents.append(['{0} ({1})'.format(driver_name, module_name), '', ])
-table_contents.append(['Driver Version Tested Against', config['last_tested_version']])
+table_contents.append(['Driver Version Tested Against', config['latest_runtime_version_tested_against']])
+table_contents.append(['Driver API Version', config['api_version']])
```

Comment:
I wish changes like these, which are independent of the main prupose of the PR, had been pushed in their own PR.

Reply:
That would have required a bunch more work:
* Manually update all old metadata to new format
* Consume new format
* Update metadata with generated metadata that also uses new format